src/presentation/api.interface.ts

```diff
@@ -1,11 +1,9 @@
-import type { DomainServices } from '@domain/index.js';
-
 /**
  * API interface
  */
-export default interface API {
+export default interface Api {
   /**
    * Runs API module
    */
-  run(domainServices: DomainServices): Promise<void>;
+  run(): Promise<void>;
 }
```

Comment:
do we need to add "init" here as well?

Reply:
no, interface describes instance methods. `init()` is static, it's not an instance method

Reply:
So how the application should know about `init()` method? It should not know about implementation details and work only with this interface

Reply:
made `init()` method non-static